src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/TokenBasedFormattingRule.cs

```diff
@@ -322,6 +322,12 @@ private static SyntaxList<UsingDirectiveSyntax> GetUsings(SyntaxNode node)
             // some * "(" cases
             if (currentToken.Kind() == SyntaxKind.OpenParenToken)
             {
+                if (previousToken.Kind() == SyntaxKind.NewKeyword &&
+                    previousToken.Parent.IsKind(SyntaxKind.MethodDeclaration, SyntaxKind.PropertyDeclaration))
+                {
+                    return CreateAdjustSpacesOperation(1, AdjustSpacesOption.ForceSpacesIfOnSingleLine);
```

Comment:
should be for more just property decls (for example, it could be a method return type). 

 it's more about if hte `new` is in a modifier list, or is in an expr.  In the latter there would be no space.  But if it's not the `new` of an object/implicit-object expr, then we should have a space.

Reply:
> should be for more just property decls (for example, it could be a method return type).

Maybe I misunderstood your comment, but I'm also testing whether we're in a method decl. I couldn't think of some other declarations that could involve `new` and a tuple type...

Reply:
things like indexers could also hit this.  I def would not try to enumerate it that way.  rather, i would have a space always *unless* it was the case we know we don't want a space (an implicit object creation).

Reply:
No, I found it above. I think this change is fine once `IndexerDeclaration` is added.

Reply:
If target typed new and new constraint is handled then I think we can just add the space.

Reply:
The `if` below handles the `new()` constraint at least (so can't be removed).
I've pushed a commit to use a more general check to handle all modifiers in declarations.